apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -1683,16 +1673,16 @@ We will be adding server redundancy to prevent lost messages.</source>
         <target>Ограничения</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
-      <trans-unit id="Local name" xml:space="preserve">
-        <source>Local name</source>
-        <target>Локальное имя</target>
-        <note>No comment provided by engineer.</note>
-      </trans-unit>
       <trans-unit id="Make a private connection" xml:space="preserve">
         <source>Make a private connection</source>
         <target>Добавьте контакт</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Make sure SMP server addresses are in correct format, line separated and are not duplicated (%@)." xml:space="preserve">
+        <source>Make sure SMP server addresses are in correct format, line separated and are not duplicated (%@).</source>
+        <target>Пожалуйста, проверьте, что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется (%@).</target>
```

Comment:
this seems like an outdated error message.. have it missed an alert somewhere?

Reply:
no, there is an alert, just in case